harness/determined/common/experimental/experiment.py

```diff
@@ -289,3 +304,11 @@ def key(ckpt: checkpoint.Checkpoint) -> Any:
 
     def __repr__(self) -> str:
         return "Experiment(id={})".format(self.id)
+
+    @classmethod
+    def _from_bindings(
+        cls, exp: bindings.v1Experiment, session: api.Session
+    ) -> "ExperimentReference":
+        exp_cls = cls(session=session, experiment_id=exp.id)
```

Comment:
Naming here is confusing for me. Since `exp_cls` is an object of type `ExperimentRef` and not the class `ExperimentRef` itself, could you please `s/exp/exp_bindings/` and `s/exp_cls/exp/` or something similar?

Reply:
good idea. renamed.